cmake-modules/CodeCoverage.cmake

```diff
@@ -156,7 +156,7 @@ function(SETUP_TARGET_FOR_COVERAGE_LCOV)
         COMMAND ${LCOV_PATH} --gcov-tool ${GCOV_PATH} -c -i -d . -o ${Coverage_NAME}.base
 
         # Run tests
-        COMMAND ${Coverage_EXECUTABLE}
+        COMMAND ${Coverage_EXECUTABLE} ${Coverage_EXECUTABLE_ARGS}
```

Comment:
Curious why is this required?

Reply:
`set(CTEST_EXTRA_ARGS -T Test --no-compress-output)`
`setup_target_for_coverage_lcov(NAME coverage EXECUTABLE ${CMAKE_CTEST_COMMAND} ${CTEST_EXTRA_ARGS} USES_TERMINAL)`
from `aktualizr/tests/CMakeLists.txt` the `Test.xml` is only generated when this argument is set.